animatch/Screens/UIKitDownloadPage/DownloadPageView.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DownloadPageViewWrapper.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 13.01.2025.
+//
+
+import SwiftUI
+
+struct DownloadPageView: UIViewControllerRepresentable {
+    let viewModel: DownloadPageViewModel
+    let startScreenViewModel: StartScreenViewModel
+
+    func makeUIViewController(context: Context) -> DownloadPageViewController {
+        return DownloadPageViewController(viewModel: viewModel, startScreenViewModel: startScreenViewModel)
```

Comment:
Ты всю найстройку viewmodel можешь уже тут сделать

```suggestion
        viewModel.setStartScreenViewModel(startScreenViewModel)
        return DownloadPageViewController(viewModel: viewModel)
```